templates/app/webpack-default/template/src/profile.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * This example demonstrates how a simple profile view can be created with
+ * Titanium. The profile includes
+ *
+ * - Profile header
+ * - A styled TableView with items that open new windows
+ * - ScrollView to allow scrolling of the whole view on smaller devices
+ */
+
+/* global tabGroup */
+
+import { createWindow, createTab, faIcon } from './utils';
+
+const title = 'Profile';
+const window = createWindow({ title, layout: 'vertical' });
+
+const scrollView = Ti.UI.createScrollView({ layout: 'vertical' });
+window.add(scrollView);
+
+// header profile pic & username
+const profilePic = Ti.UI.createImageView({
+	image: 'images/profile.jpg',
+	top: 30,
+	width: 120,
+	height: 120,
+	borderRadius: 60
+});
+scrollView.add(profilePic);
+const username = Ti.UI.createLabel({
+	text: 'Verona Blair',
+	font: {
+		fontSize: 28
+	},
+	top: 20
+});
+scrollView.add(username);
+
+// profile sub items table view
+const items = [
+	{ title: 'Personal Details', icon: 'user' },
+	{ title: 'Privacy & Security', icon: 'lock' },
+	{ title: 'Legal', icon: 'balance-scale-left' }
+];
+
+const settingsSection = Ti.UI.createTableViewSection({
+	headerView: createSeparator(),
+	footerView: createSeparator()
+});
+items.forEach(item => settingsSection.add(createRow({ ...item })));
+const logoutSection = Ti.UI.createTableViewSection();
+logoutSection.add(createRow({
+	title: 'Logout',
+	icon: 'sign-out-alt',
+	titleTint: '#E82C2A',
+	iconTint: '#E82C2A',
+	iconBgTint: '#FFC5C3'
+}));
+const tableView = Ti.UI.createTableView({
+	data: [ settingsSection, logoutSection ],
+	top: 40,
+	height: Ti.UI.SIZE,
+	scrollable: false,
+	rowSeparatorInsets: { left: 20, right: 20 },
+	separatorStyle: Ti.UI.TABLE_VIEW_SEPARATOR_STYLE_NONE
```

Comment:
You're setting the `rowSeparatorInsets` property to shorten the line width, but you're also removing the line by setting `separatorStyle` to NONE.

Reply:
Ah good catch. I probably missed that since i initially had separators but then removed them since i liked it better without.